src/components/global/Playground/index.tsx

```diff
@@ -264,16 +346,16 @@ const FRAME_SIZES = {
   xlarge: '800px',
 };
 
-const waitForFrame = (frame: HTMLElement) => {
+const waitForFrame = (frame: HTMLIFrameElement) => {
   if (isFrameReady(frame)) return Promise.resolve();
 
-  return new Promise((resolve) => {
+  return new Promise<void>((resolve) => {
     frame.contentWindow.addEventListener('demoReady', () => {
       resolve();
     });
   });
 };
 
-const isFrameReady = (frame: HTMLElement) => {
-  return frame.contentWindow.demoReady === true;
+const isFrameReady = (frame: HTMLIFrameElement) => {
+  return (frame.contentWindow as any).demoReady === true;
```

Comment:
Why do we need to cast this as `any`?

Reply:
`demoReady` is a made up prop that we are directly assigning to `window`. The type of `frame.contentWindow` (`Window`) won't include that value and will throw a type error in an IDE that does type checking.